Formula/a/abseil.rb

```diff
@@ -55,17 +43,10 @@ def install
                     *extra_cmake_args, *std_cmake_args
     system "cmake", "--build", "build"
     system "cmake", "--install", "build"
-
-    return unless OS.mac?
-
-    # Remove bad flags in .pc files.
-    # https://github.com/abseil/abseil-cpp/issues/1408
-    inreplace lib.glob("pkgconfig/absl_random_internal_randen_hwaes{,_impl}.pc"),
-              "-Xarch_x86_64 -Xarch_x86_64 -Xarch_arm64 ", ""
```

Comment:
Fix is supposedly merged upstream but we're still seeing similar issues, e.g., in `brew test grpc` or `brew install mosh`. @ZhongRuoyu, mind having a look?

Reply:
This was fixed by https://github.com/abseil/abseil-cpp/pull/1412 but then broken by https://github.com/abseil/abseil-cpp/pull/1707. Now `Cflags` in `lib/pkgconfig/absl_random_internal_randen_hwaes.pc` contains: `SHELL:-Xarch_x86_64 -msse4.1 SHELL:-Xarch_arm64 -march=armv8-a+crypto`.

Reply:
Fix proposed in https://github.com/abseil/abseil-cpp/pull/1738. Pushed an update to include the patch (and rebase against `protobuf` updates).

Reply:
Thanks @ZhongRuoyu!